frontend/public/components/utils/about-modal.tsx

```diff
@@ -0,0 +1,54 @@
+import * as React from 'react';
+import * as classNames from 'classnames';
+import { AboutModal as PFAboutModal } from 'patternfly-react';
+
+import { k8sVersion } from '../../module/status';
+import { getBrandingDetails } from '../masthead';
+
+export class AboutModal extends React.Component<AboutModalProps, AboutModalState> {
+  /* eslint-disable no-undef */
+  state = {
+    showAboutModal: false,
+    kubernetesVersion: null,
+  };
+
+  componentDidMount() {
+    this.checkKubernetesVersion();
+    this.setState({ showAboutModal: true });
```

Comment:
Why are we setting `showAboutModal` to `true` in this method rather than when we initialize `state` above?